platform/lib/format-transform/index.test.js

```diff
@@ -13,6 +13,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+const {join} = require('path');
+const {BUILD} = require('@lib/utils/project').paths;
+
+// Use the downloaded validator.js:
+const VALIDATOR_TARGET_PATH = join(BUILD, 'testing/validator.js');
```

Comment:
Can we add a check that'll test if the file exists and prints a warning if it doesn't?